config/routes.config.js

```diff
@@ -5,6 +5,7 @@ const routes = [
   { path: '/offramp', editInfo: 'skip' },
   { path: '/login/sin' },
   { path: '/login/dateOfBirth' },
+  { path: '/login/error/doesNotMatch', editInfo: 'login.error'},
```

Comment:
I added the editInfo here to a variable that doesn't exist - my reasoning is to ensure that this page doesn't show up when pressing "back" from the next page, but to allow it to stay here in the flow. If there's a better way to do this, happy to change it :)

Reply:
We've done this in the past by using 'skip' (you can see above). It's doing what you are doing here, but it's easier to see which pages are skipped.